backend/fleet_management/api.py

```diff
@@ -1,11 +1,26 @@
-from rest_framework import views
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import generics, filters, views
 from rest_framework.response import Response
-
-from .serializers import UserSerializer
+from .serializers import CarSerializer, UserSerializer
+from .models import Car
+from rest_framework.permissions import IsAuthenticated
 
 
 class CurrentUserRetrieveView(views.APIView):
     def get(self, request):
         return Response(
             UserSerializer(request.user).data
         )
+
+
+class CarListView(generics.ListAPIView):
+    authentication_classes = (IsAuthenticated,)
+    serializer_class = CarSerializer
+    queryset = Car.objects.all()
+    filter_backends = (filters.OrderingFilter, filters.SearchFilter, DjangoFilterBackend,)
+    search_fields = (
```

Comment:
I think that if we would have to search cars then the only field we will search by would be plates. However, it does not break anything so let's not close any 🚪 :)